modules/fluid_properties/include/fluidproperties/VaporMixtureFluidProperties.h

```diff
@@ -113,7 +128,8 @@ class VaporMixtureFluidProperties : public FluidProperties
   propfunc(p, v, e)
   propfunc(T, v, e)
   propfunc(c, v, e)
-  propfunc(rho, p, T)
+  propfunc(v, p, T)
+  propfunc(rho, p, T) // TODO: deprecate this one in favor of v(p,T)
```

Comment:
```suggestion
  propfunc(rho, p, T)
```
we dont use v in NS we like rho